core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -2115,12 +2115,12 @@ public <T> T act(ProcessCallable<T> callable) throws IOException, InterruptedExc
     }
 
     /*
-        On MacOS X, there's no procfs <http://www.osxbook.com/book/bonus/chapter11/procfs/>
-        instead you'd do it with the sysctl <http://search.cpan.org/src/DURIST/Proc-ProcessTable-0.42/os/darwin.c>
-        <http://developer.apple.com/documentation/Darwin/Reference/ManPages/man3/sysctl.3.html>
+        On MacOS X, there's no procfs <https://web.archive.org/web/20200513034043/https://osxbook.com/book/bonus/chapter11/procfs/>
```

Comment:
Alternative reference, but I doubt it's as useful 😁 

```suggestion
        On MacOS X, there's no procfs (Amit Singh, Mac OS X Internals: A Systems Approach (Addison-Wesley Professional, 2006), page 1409)>
```